mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -316,6 +349,62 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
         }
     }
 
+    const handleAgentResponse = (
```

Comment:
After building the plan, if I refresh my browser, the chat plan of attack is not dispalyed properly, I just get the json back. 

I think all we need to do is call the `handleAgentReponse` in the fetchHistory function when the message we get back is of type agent:planning. 

ie: we should update this code 

```typescript
// 4. Add messages to the ChatHistoryManager
history.forEach(item => {
    newChatHistoryManager.addChatMessageFromHistory(item);
});
```

<img width="357" alt="Screenshot 2025-02-10 at 1 34 08 PM" src="https://github.com/user-attachments/assets/adda096a-8e67-4856-8036-03ea7cb1b65f" />

Reply:
This won't add back the LetsGo button, but let's ignore that edge case for now. 

So: we want to render all of the planning messages. They won't be purple, but that's okay for now.